edb/server/protocol/system_api.py

```diff
@@ -89,8 +89,13 @@ def _response_ok(response, message):
 
 
 async def _ping(server):
+    if server.get_backend_runtime_params().has_create_database:
+        dbname = edbdef.EDGEDB_SYSTEM_DB
+    else:
+        dbname = edbdef.EDGEDB_SUPERUSER_DB
```

Comment:
Will it be adjusted to the database name provided in BACKEND_DSN? i.e. if the (one and only) available database is named differently (not `edgedb`), will it still work? 

https://github.com/edgedb/edgedb/issues/5592#issuecomment-1577786582
> In single db mode EdgeDB uses the database provided in --backend-dsn (EDGEDB_SERVER_BACKEND_DSN), e.g. if you give it postgres://host:port/mydb it'll use mydb.

Reply:
Yes. In single-database mode `edgedb` gets translated to whatever was specified in the backend DSN.